Makefile

```diff
@@ -0,0 +1,64 @@
+TARGET=oscar
```

Comment:
This MAKEFILE is tested and works correctly, right?

Also might be worth adding an `all` target